avm/res/kubernetes-configuration/flux-configuration/main.bicep

```diff
@@ -69,7 +69,7 @@ resource managedCluster 'Microsoft.ContainerService/managedClusters@2022-07-01'
   name: clusterName
 }
 
-resource fluxConfiguration 'Microsoft.KubernetesConfiguration/fluxConfigurations@2022-03-01' = {
+resource fluxConfiguration 'Microsoft.KubernetesConfiguration/fluxConfigurations@2023-05-01' = {
```

Comment:
What about versioning? As per guidelines an API version update should lead to a patch bump, but is that the case also in this scenario for which the API version update allows a new feature?

Reply:
Good question @eriqua. Maybe 😂
The thing is, we don't even expose additional properties and from the description it sounded like it's essentially fixing a bug.
I would however lean toward an 0.2 regardless, I guess.

Reply:
Updated to 0.2

Reply:
Works both ways for me, honestly I'm also undecided. That was more a question. Please resolve this comment as you see fit.